src/soft_start.cpp

```diff
@@ -6,35 +6,62 @@
 
 
 #include "kodlab_mjbots_sdk/soft_start.h"
+#include "kodlab_mjbots_sdk/log.h"
 
 namespace kodlab {
 
-void SoftStart::Constrain(std::vector<float> &values,
+void TorqueLimiter::Constrain(std::vector<float> &values,
                           float min_val,
                           float max_val) {
   for (auto &value : values) {
-    value = fmin(fmax(value, min_val), max_val);
+    value = std::min(std::max(value, min_val), max_val);
   }
 }
 
-float SoftStart::Constrain(float values, float min_val, float max_val) {
-  return fmin(fmax(values, min_val), max_val);
+float TorqueLimiter::Constrain(float values, float min_val, float max_val) {
+  return std::min(std::max(values, min_val), max_val);
```

Comment:
Should be value I think, unless it's useful in the Overload ide pop ups

Reply:
done.